### DISCUSSION TITLE
잔여 백신 종류 선택 관련해서 질문 드립니다.

### DISCUSSION THREAD

**Discussion by ZER0ADD** [kimytsc/covid-rest-vaccine-macro#80] [Q&A] [answered]
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=used&x=127.1054288&y=37.3594909&bounds=127.1022772%3B37.3577853%3B127.1085804%3B37.3611964",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",/*화이자, 2021-08-17 기준 18세 이상(2003.12.31 이전 출생자)만 신청 가능*/
        "VEN00014",/*모더나, 2021-08-17 기준 18세 이상(2003.12.31 이전 출생자)만 신청 가능*/
      /*"VEN00015",/*아스트라제네카, 2021-08-17 기준 30세 이상(1991.12.31 이전 출생자)만 신청 가능하여 주석처리, 필요시 '/*"VEN00015'의 '/*' 제거*/
      /*"VEN00016",/*얀센, 기본값에서 제거, 필요시 '/*"VEN00016'의 '/*' 제거*/
      /*"VEN00017",/*노바백스, 기본값에서 제거, 필요시 '/*"VEN00017'의 '/*' 제거*/
      /*"VEN00018",/*시노팜, 기본값에서 제거, 필요시 '/*"VEN00018'의 '/*' 제거*/
      /*"VEN00019",/*시노백, 기본값에서 제거, 필요시 '/*"VEN00019'의 '/*' 제거*/
      /*"VEN00020",/*스푸트니크V, 기본값에서 제거, 필요시 '/*"VEN00020'의 '/*' 제거*/
    ]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/naver/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()

올려주신 소스에서 화이자와 모더나를 제외하고 다 제거하고 싶은데 어떻게 코드를 수정해야하는지 정확히 이해가 가지 않아 질문드립니다. 혹시 예시를 보여주실수 있을까요?

**Comment by kimytsc** [comment, marked as answer]
현재 기본값으로 화이자와 모더나만 선택되어 있는 상태입니다. 

`choice: []`: 모든 잔여백신 선택
`choice: ["VEN00013","VEN00014"]`: 화이자, 모더나만 선택
`choice: ["VEN00014","VEN00015"]`: 모더나, AZ만 선택

필요에 따라 이렇게 수정해서 쓰셔도 됩니다.

**Reply by ZER0ADD** [reply]
감사합니다.
